### PR TITLE
Update ReadMe to include links to pricing and billing

### PR DESCRIPTION
**Pull request by ecfan**
Based on customer feedback that said they weren't given warning that integration accounts incur costs. I also updated the grammar and formatting. However, screenshots need updates to the current version as they have changed dramatically.

# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Updated ReadMe to include note that integration accounts incur costs
* Revised for grammar, terminology, and formatting according to Microsoft style
* Updated metadata with today's date and other grammar edits

### CODE REVIEW

File: 201-logic-app-as2-send-receive/README.md [hunk 1]
```diff
@@ -13,35 +13,59 @@
...
 [![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-logic-app-as2-send-receive%2Fazuredeploy.json)
 
-This template demonstrates AS2 Send Receive using Logic Apps. It creates Integration Accounts for two partners (Contoso and Fabrikam) and adds Partners and Agreements into them. It creates Logic Apps between Fabrikam Sales and Contoso which demonstrate Sync AS2 Send Receive. It also creates Logic Apps between Fabrikam Finance and Contoso which demonstrate ASync AS2 Send Receive.
+This template creates an AS2 Send Receive workflow by using Azure Logic Apps to create the following Azure resources:
+
+* Integration accounts for two partners, Contoso and Fabrikam, along with the necessary artifacts, partners and agreements.
+* Logic apps between Fabrikam Sales and Contoso, which demonstrate Sync AS2 Send Receive.
+* Logic apps between Fabrikam Finance and Contoso, which demonstrate ASync AS2 Send Receive.
+
+> [!IMPORTANT]
```
Comment (bmoore-msft) [inline 201-logic-app-as2-send-receive/README.md:22]: this is not valid markdown

File: 201-logic-app-as2-send-receive/metadata.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
   "type": "QuickStart",
     "itemDisplayName": "Azure Logic Apps - AS2 Send Receive",
-    "description": "Creates Integration Accounts for two partners (Contoso and Fabrikam) and adds Partners and Agreements into them. It creates Logic Apps between Fabrikam Sales and Contoso which demonstrate synchronous AS2 Send Receive. It also creates Logic Apps between Fabrikam Finance and Contoso which demonstrate asynchronous AS2 Send Receive.",
-    "summary": "Demonstrates AS2 Send Receive using Logic Apps",
+    "description": "Creates integration accounts for two partners, Contoso and Fabrikam, including the artifacts for partners and agreements. The template also creates logic apps between Fabrikam Sales and Contoso to demonstrate synchronous AS2 Send Receive. The template also creates logic apps between Fabrikam Finance and Contoso, which demonstrate asynchronous AS2 Send Receive.",
+    "summary": "Demonstrates AS2 Send Receive scenario using Azure Logic Apps",
```
Comment (bmoore-msft) [inline 201-logic-app-as2-send-receive/metadata.json:6]: update the dateUpdated as welll

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 201-logic-app-as2-send-receive/README.md (hunk 1 above)
Comment (ecfan) [reply 201-logic-app-as2-send-receive/README.md:24]: Replaced asterisks with dashes, thanks!

File: 201-logic-app-as2-send-receive/metadata.json (hunk 2 above)
Comment (ecfan) [reply 201-logic-app-as2-send-receive/metadata.json:6]: Done, thanks!

**Comment by ecfan** [comment]
#sign-off

**Comment by ecfan** [comment]
Hi @bmoore-msft, I resolved all the issues. Please review?

File: 201-logic-app-as2-send-receive/README.md (hunk 1 above)
Comment (bmoore-msft) [reply 201-logic-app-as2-send-receive/README.md:24]: It's not the bullets - it's the 

[!IMPORTANT]

syntax, not sure what you want that to do, but in github it appears literally.

Comment (ecfan) [reply 201-logic-app-as2-send-receive/README.md]: @bmoore-msft, thanks I fixed the formatting on those notes. In our doc repo, that note style is supported.

**Comment by ecfan** [comment]
Hi @bmoore-msft, I fixed the formatting issues.

**Comment by ecfan** [comment]
@bmoore-msft, thank you!

**Pull request merged**